### PR TITLE
Implement enter checkup

### PR DESCRIPTION
**Pull request by kmitche9**

### CODE REVIEW

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (alexofceaser85) [inline CS3230Project/DAL/Nurses/NurseDal.cs]: What is this, looks like this  is a temporary thing that should be removed.

Comment (alexofceaser85) [reply CS3230Project/DAL/Nurses/NurseDal.cs]: Looking at this more we cannot just set the userName to 2

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (alexofceaser85) [inline CS3230Project/DAL/Nurses/NurseDal.cs:44]: There are some ordinals that are not used. They should be removed

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (alexofceaser85) [inline CS3230Project/DAL/Nurses/NurseDal.cs]: This should either be removed or changed to an actual username value

File: CS3230Project/DAL/Accounts/AccountsDal.cs
Comment (alexofceaser85) [inline CS3230Project/DAL/Accounts/AccountsDal.cs:50]: We should not be using '*' to select the rows, that was said by Yang in class. This should be reverted to specify all the values for nurse. I think we have this issue in more DAL files though so maybe we should think about making a separate issue for this later.

File: CS3230Project/Model/Users/Nurses/Nurse.cs
Comment (alexofceaser85) [inline CS3230Project/Model/Users/Nurses/Nurse.cs]: There is a constant in the settings class for the maximum lengths

File: CS3230Project/View/SearchPatient.cs
Comment (alexofceaser85) [inline CS3230Project/View/SearchPatient.cs]: What exactly is this "d" here?

**Review by alexofceaser85 — CHANGES_REQUESTED** [review]

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Nurses/NurseDal.cs]: This is a temp thing. It will be removed when I finish the Nurse class. I had planned to do that after the merge.

Comment (kmitche9) [reply CS3230Project/DAL/Nurses/NurseDal.cs]: I can make those changes now that the branches are merged.

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Nurses/NurseDal.cs:44]: I will fix this. I was waiting on the merge.

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Nurses/NurseDal.cs]: Will be fixed. Waiting on merge.

File: CS3230Project/DAL/Accounts/AccountsDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Accounts/AccountsDal.cs:50]: It will be updated.

File: CS3230Project/Model/Users/Nurses/Nurse.cs
Comment (kmitche9) [reply CS3230Project/Model/Users/Nurses/Nurse.cs]: Will be fixed.

File: CS3230Project/View/SearchPatient.cs
Comment (kmitche9) [reply CS3230Project/View/SearchPatient.cs]: it is for the date format. I will check into it and see if it is needed or incorrect format.

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Nurses/NurseDal.cs]: Fixed

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Nurses/NurseDal.cs:44]: Fixed. 

Also, the GetNurseByID() method is not currently being used. I was thinking wrong while implementing something else, and by the time I had realized, I was done with the query and method. It can be removed, but I left it for now just in case we could use it in the near future.

File: CS3230Project/DAL/Nurses/NurseDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Nurses/NurseDal.cs]: fixed

File: CS3230Project/DAL/Accounts/AccountsDal.cs
Comment (kmitche9) [reply CS3230Project/DAL/Accounts/AccountsDal.cs:50]: I will take a look at this and try to update the query. I agree, we probably should plan to review all of our queries.

File: CS3230Project/Model/Users/Nurses/Nurse.cs
Comment (kmitche9) [reply CS3230Project/Model/Users/Nurses/Nurse.cs]: Fixed.

File: CS3230Project/View/SearchPatient.cs
Comment (kmitche9) [reply CS3230Project/View/SearchPatient.cs]: I changed it to "MM/dd/yyyy", but "d" does the exact same thing. I suppose it is more readable with the actual format.

File: CS3230Project/View/Checkup.cs
Comment (alexofceaser85) [inline CS3230Project/View/Checkup.cs:118]: I would be nice to extract this into a field, I don't know what 2 is supposed to mean,.

**Review by alexofceaser85 — CHANGES_REQUESTED** [review]

File: CS3230Project/View/Checkup.cs
Comment (kmitche9) [reply CS3230Project/View/Checkup.cs:118]: I will fix it.

**Review by alexofceaser85 — APPROVED** [review]

**Pull request merged**
